engine/routines/checkpokerus.asm

```diff
@@ -0,0 +1,25 @@
+CheckPokerus: ; 4d860
```

Comment:
I think this one can just be engine/check_pokerus.asm. Even though it's small it's more visible, and it's often the first thing people look for.

Reply:
Moved it to engine/pokerus/ along with the other Pokérus-related engine files. We can probably do the same for other engine/ subsystems.